utils/build/docker/nodejs/express4-typescript/rasp.ts

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+import type { Express, Request, Response } from 'express';
+
+const http = require('http')
+function initRaspEndpoints (app: Express) {
+    app.get('/rasp/ssrf', (req: Request, res: Response) => {
+        const cl = http.get(`http://${req.query.domain}`, () => {
```

Comment:
Can we rework this naming? http.get returns a `ClientRequest` and `cl` is a bit confusing to me.

Reply:
oh, I thought that I was using `cr` 🤦 
But yes, I will use `clientRequest` instead.